docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,112 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
+
+- Device name: `Nokia G-240W-B`
+- Vendor: `Nokia`
+- Hardware version: `3FE56756ADBA`
+- Software version: `3FE47969AGCA21`
+- Chipset: `BCM6838`
+
+> Note:
+There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+- The `Raspberry Pi` is using static IP address, `IPv4` and `IPv6` as well.
+- The `ISP` provides `IPv6` address, prefix and gateway to the router.
+- `Pi-hole` has enabled `DHCP server`.
+- `Pi-hole` has enabled `IPv6 support (SLAAC + RA)`.
+
+For this example, we will be using the `192.168.1.100` and `2806:103e:1f:face::1681:100` addresses for the following steps. You will need to substitute your own addresses in the following steps.
```

Comment:
Why do you recommend a global address here? I can change if your ISP assigned a different prefix We usually recommend a ULA address.

https://discourse.pi-hole.net/t/use-ipv6-ula-addresses-for-pi-hole/2127

Reply:
Since this Router doesn't have a section for ULA (explicitly) I configured with IPV6 SLAAC however I realized after reading your comment that the Router offers ULA if the DHCP6 is enabled.

Reply:
I restarted the router and the IPV6 prefix has been updated, however the communication is over the link-local IP.